04/app.js

```diff
@@ -1,9 +1,28 @@
 const user = {
-    firstName: 'Adam',
-    lastName: 'Nowak',
-    born: {
-        day: '14',
-        month: '04',
-        year: '1985'
-    }
-}
\ No newline at end of file
+  firstName: "Adam",
+  lastName: "Nowak",
+  born: {
+    day: "08",
+    month: "04",
+    year: "1985",
+  },
+};
+
+const now = new Date();
+const currentDay = now.getDate();
+const currentMonth = now.getMonth();
+
+const day = Number(user.born.day);
+const month = Number(user.born.month);
+
+birthdayCheck();
+
+function birthdayCheck() {
+  if (day === currentDay && month === currentMonth) {
```

Comment:
Sprawdziłaś czy działa? `now.getMonth();` zwraca miesiąc o jeden numer mniej więc raczej nie działa to jak należy ;)